### ISSUE TITLE
[4.1.16]  Release build in Xcode hangs with barebones Expo and NativeWind project

### ISSUE THREAD

**Issue by loksland** [nativewind/nativewind#1143]
Thanks for providing this great library, I am running into an issue trying to build an iOS release:

**Describe the bug**
- Release build fails in Xcode with a bare bones Expo project 
- Build hangs at `Bundle React Native code and images` step

**Reproduction**
- Create a new Expo project with `npx create-expo-app@latest` ("expo": "~51.0.28")
- Add native wind v4 as per instructions (https://www.nativewind.dev/v4/getting-started/expo-router)
  - Specifies to use `nativewind@^4.0.1` which installs v4.1.16
  - Add `import "../global.css"` to `/app/_layout.tsx`
- Build ios with `npx expo prebuild --platform ios --clean --npm`
- Open in Xcode and build a release
- Build gets to `Bundle React Native code and images` step and hangs indefinitely
- Difficult to work out what is going on as no error message is displayed
- I've created a test repo with the issue here for reference: https://github.com/loksland/nativewindexpotest

**Expected behaviour**
Successful build

**Notes**
- Commenting out `import "../global.css"` in `/app/_layout.tsx` builds successfully
- Development build with `npx expo run:ios` works fine
- Tried adding simlinks below as recommended in similar sounding issue (https://github.com/nativewind/nativewind/issues/641) though this did not fix
```bash
sudo ln -s $(which npx) /usr/local/bin/npx
sudo ln -s $(which node) /usr/local/bin/node
```

**Additional context**
- M1 Mac environment with Sonoma 14.5 (23F79) 
- XCode Version 15.4 (15F31d)
- This hanging behaviour appears to be also present with EAS (remote and local) builds though still yet to confirm this as it was in a separate project
- Should I be using a different version of Native Wind?
- Would appreciate any guidance to be able to debug the issue or a workaround

**Comment by loksland** [comment]
Just an update that @ivstiv's described method described here (`NATIVEWIND_WATCH: "false"`) appears to fix the issue for me:
https://github.com/nativewind/nativewind/issues/1139#issuecomment-2414985298

**Comment by loksland** [comment]
Trying v4.1.10 I get an XCode error:
`
Starting Metro Bundler
warning: Bundler cache is empty, rebuilding (this may take a minute)
env: node: No such file or directory
error: Command failed: /Users/pathtoproject/nativewindexpotest/node_modules/tailwindcss/lib/cli.js --input /Users/pathtoproject/nativewindexpotest/global.css --output /Users/pathtoproject/nativewindexpotest/node_modules/nativewind/.cache/global.css.ios.css
env: node: No such file or directory
`
- Can confirm the input `global.css` is present at that path

**Comment by SuzuyaBish** [comment]
Same issue with me, is there any way around this?

**Comment by marklawlor** [comment]
Fixed in `4.1.18`

**Comment by loksland** [comment]
Thanks, can confirm `4.1.18` fixes the issue.
